stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/ChannelItem.kt

```diff
@@ -97,11 +100,13 @@ public fun ChannelItem(
     },
 ) {
     val channel = channelItem.channel
+    val description = stringResource(id = R.string.stream_compose_cd_channel_item)
 
     Column(
         modifier = modifier
             .fillMaxWidth()
```

Comment:
Basically, there are 2 ways to identify a component in the tree: either by content description or by using a modifier with test tag. Seems that using content description is a better approach as also fix accessibility as a part of the effort.